src/inference/Inference.jl

```diff
@@ -517,24 +517,24 @@ function predict(model::Model, chain::MCMCChains.Chains; kwargs...)
     return predict(Random.GLOBAL_RNG, model, chain; kwargs...)
 end
 function predict(rng::AbstractRNG, model::Model, chain::MCMCChains.Chains; include_all = false)
+    # Don't need all the diagnostics
+    chain = MCMCChains.get_sections(chain, :parameters)
```

Comment:
Can we maybe use different variable names here? It can be a bit tricky, both for the compiler and reader, if different objects of possibly different type share the same name.

Reply:
Good point :+1: